aws-lambda-layer/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -x
```

Comment:
Is this file now irrelevant with the introduction of the Rakefile?

Reply:
Not anymore, I deleted it.